terraform/_account/main.tf

```diff
@@ -0,0 +1,11 @@
+module "common_resources" {
```

Comment:
no need to have submodules /terraform/_common and /modules/_account

instead the content of /terraform/_account should be go into root / and content of /modules/_common should be go into root as some .tf files